### PR TITLE
enable addtional security groups for rosa-sts-advanced

### PR DESCRIPTION
**Pull request by xueli181114**

### CODE REVIEW

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-e2e-f3

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by xueli181114** [comment]
Blocked due to [OCM-4506](https://issues.redhat.com/browse/OCM-4506)
Will retrigger the job after this issue fixed.

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by xueli181114** [comment]
[OCM-4506](https://issues.redhat.com/browse/OCM-4506) is fixed. Re-run now.

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-e2e-f3

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-e2e-f3

**Comment by xueli181114** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-e2e-f3

File: ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh [hunk 1]
```diff
@@ -339,6 +340,13 @@ if [[ "$ENABLE_BYOVPC" == "true" ]]; then
     record_cluster "subnets" "public_subnet_ids" ${PUBLIC_SUBNET_IDs}
   fi
 fi
+# Additional security groups options
+SECURITY_GROUP_ID_SWITCH=""
+if [[ "$ADDITIONAL_SECURITY_GROUP" == "true" ]]; then
+  SECURITY_GROUP_IDs=$(cat ${SHARED_DIR}/security_groups_ids | xargs|sed 's/ /,/g')
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh:346]: Is `xargs` here needed? And miss a `space` after it.

File: ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh [hunk 2]
```diff
@@ -339,6 +340,13 @@ if [[ "$ENABLE_BYOVPC" == "true" ]]; then
...
+  SECURITY_GROUP_ID_SWITCH=" --additional-compute-security-group-ids ${SECURITY_GROUP_IDs}"
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh:347]: A redundant `space` before `--`.

**Comment by xueli181114** [comment]
@liangxia Can you please review?

**Pull request merged**